tests/Gedmo/Mapping/Fixture/Loggable.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Mapping\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity
+ * @Gedmo\Loggable
+ */
+class Loggable
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
```

Comment:
Please, use type hints and type declarations where possible (in this and the remaining classes / methods introduced by this PR).